Sources/SIL/SIL.swift

```diff
@@ -495,6 +495,7 @@ public enum TupleElements: Equatable {
 public indirect enum Type: Equatable {
     case addressType(_ type: Type)
     case attributedType(_ attributes: [TypeAttribute], _ type: Type)
+    case coroutineTokenType
```

Comment:
I had to create this, because the coroutine tokens never appear annotated with a type (they can only be results of `begin_apply` and sole arguments to `end_apply`). However, it's convenient to be able to wrap them into `Operand`s, so this is a placeholder type for them.